include/seqan3/range/views/minimiser.hpp

```diff
@@ -391,12 +478,51 @@ class minimiser_view<urng_t>::window_iterator
 
         return false;
     }
+
+    //!\brief Calculates the next minimiser value when two ranges are given.
+    // For the following windows, we remove the first window value (is now not in window_values) and add the new
+    // value that results from the window shifting.
+    bool next_minimiser()
+    //!\cond
+        requires urange2_is_given
+    //!\endcond
+    {
+        std::ranges::advance(window_right, 1);
+        std::ranges::advance(window_right2, 1);
+        if (window_right == urange_end)
+            return true;
+
+        uint64_t new_value = *window_right;
+        if (*window_right2 < new_value)
+            new_value = *window_right2;
+
+        if (minimiser_value == *(std::begin(window_values)))
```

Comment:
```suggestion
        if (minimiser_value == window_values.front())
```
Can you check if there are other occurrences?